docs/using_starrocks/Materialized_view.md

```diff
@@ -169,13 +169,13 @@ GROUP BY order_id;
   - Strict consistency is not guaranteed between the materialized view and the base tables in the external catalog.
   - Currently, building asynchronous materialized views based on external resources is not supported.
   - Currently, StarRocks cannot perceive the data changes on the base tables in Iceberg catalogs and Hudi catalogs, so all partitions are refreshed by default every time the refreshing task is triggered. If you want to refresh only some of the partitions, you can manually refresh the materialized view using the [REFRESH MATERIALIZED VIEW](../sql-reference/sql-statements/data-manipulation/REFRESH%20MATERIALIZED%20VIEW.md) statement and specify the partition you want to refresh.
-  - From v2.5.5 onwards, StarRocks can periodically refresh the metadata of the frequently accessed Hive catalogs to perceive data changes. You can configure the Hive metadata refresh through the following FE parameters:
+  - From v2.5.5 onwards, StarRocks can periodically refresh the metadata cache of the frequently accessed Hive catalogs to perceive data changes. You can configure the Hive metadata refresh through the following FE parameters:
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/b5f52d9a-46c4-4c3e-b3de-114220dc544b)
should we say “the cached metadata of xxx" because we can't say specific external catalogs HAS a specific metadata cache?